plugins/node/opentelemetry-instrumentation-pino/src/instrumentation.ts

```diff
@@ -61,21 +63,25 @@ export class PinoInstrumentation extends InstrumentationBase {
                 args.splice(0, 0, {
                   mixin: instrumentation._getMixinFunction(),
                 });
-                return pinoModule(...args);
+                return moduleExports(...args);
               }
             }
 
             args[0] = instrumentation._combineOptions(args[0]);
 
-            return pinoModule(...args);
-          }, pinoModule);
+            return moduleExports(...args);
+          }, moduleExports);
 
           if (typeof patchedPino.pino === 'function') {
             patchedPino.pino = patchedPino;
           }
           if (typeof patchedPino.default === 'function') {
             patchedPino.default = patchedPino;
           }
+          if (isESM) {
+            module.pino = patchedPino;
+            module.default = patchedPino;
+          }
```

Comment:
> Codecov / codecov/patch
> plugins/node/opentelemetry-instrumentation-pino/src/instrumentation.ts#L82-L83
> Added lines #L82 - L83 were not covered by tests

These lines actually *are* covered by the test, but I guess the out of process `runTestFixture` doesn't catch that.